examples/chrono/chrono.cpp

```diff
@@ -0,0 +1,150 @@
+﻿/*
+ * Copyright (c) 2022, Christopher Kormanyos
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <chrono>
+
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+static void app_hw_init();
+
+auto main() -> int
```

Comment:
I still don't understand the point of this spelling in such a place.

Reply:
>  point of this spelling in such a place.

What spelling do you mean? Do you mean `int main()`?

Reply:
I guess the comment refers to the trailing return type. I am fine with either notation.

Reply:
> the trailing return type.

Well it's kind of a funny story. When I started using `clang-tidy`, the most prominent genre of messages I was getting were in the category `modernize-*`. This particular style of modern syntax (the trailing return type) was the message I did, in fact, most prominently receive.

So I went ahead and modernized everything in my style that I learned from those experiences --- thinking that would be best --- although I must admit, we're pretty used to seeing `int main()`

OK, long story, so when I get around to switching over to tabs as line indentations (see also #13), I'll go ahead and use the traditional form for `main()`.